### ISSUE TITLE
Cleanup command

### ISSUE THREAD

**Issue by nanoxd** [Carthage/Carthage#902]
Thanks for all the hard work on this project :smile:.

With the addition of generating `bcsymbolmap` files, I've seen an influx of things in my Carthage build folders. Before the introduction of the `bcsymbolmap` files, it was easy to go in there myself and delete the files. That is no longer the case with symbol maps using a UUID unless you remove the entire Carthage folder and run `carthage bootstrap`

I suggest an additional command be added to cleanup files for when a user wants to remove the files from their Carthage build/checkout folders that targets the `.framework`, `.bcsymbolmap`, and `.dSYM` artifacts.

**Comment by ffittschen** [comment]
I totally agree! Since the addition of the `bcsymbolmap` files the folder looks like a mess... The `Carthage/Build` folder is on my `.gitignore` but I still like to have things clean in the repo.

The deletion of unneeded files should be automated...

<img width="671" alt="screen shot 2016-04-14 at 10 09 53" src="https://cloud.githubusercontent.com/assets/7734806/14521363/34deffe8-0229-11e6-8b3e-f148f317314b.png">

**Comment by mdiep** [comment]
Maybe Carthage should delete old `.bcsymbolmap`s when building?

**Comment by ffittschen** [comment]
@mdiep This would be perfect 👍🏻
If you use `carthage build` without the specification of a project, all `.bcsymbolmap`s should be deleted, but if you use e.g. `carthage build Alamofire` it should just delete the corresponding `.bcsymbolmap`

**Comment by BProg** [comment]
I would also need this feature for cleanup `bcsymbolmap` files because we keep them in git, so the someone who downloads the project don't need to use `carthage` for the project to run. A `cleanup` command will be very handy

**Comment by ikesyo** [comment]
There is a ongoing PR #2333.

**Comment by ravitripathi** [comment]
Is this still being worked on?

**Comment by idanzhu** [comment]
not find out command `carthage cleanup`, it had been merged ?

**Comment by ravitripathi** [comment]
It has been merged, but carthage cleanup doesn't work as of 0.34.0.

**Comment by yourshinsuke** [comment]
@ravitripathi me too. When??

**Comment by ravitripathi** [comment]
@yourshinsuke no clue. It seems like the feature was never exposed. Weird. We ended up commit those bsymbolmaps too along with the framework files after updating.
